service/src/main/resources/application.yaml

```diff
@@ -62,6 +62,8 @@ fpl:
     mapping: 'example=>Family Court:admin@family-court.com'
   local_authority_code_to_cafcass:
     mapping: 'example=>cafcass:cafcass@cafcass.com'
+  local_authority_code_to_local_authority:
+    mapping: 'example=>Example Local Authority:local-authority@local-authority.com'
```

Comment:
It will need to be added to couple of more places, please let me know if you are close to merge but essentially:
* charts/fpl-case-service/values.aat.template.yaml
* charts/fpl-case-service/values.preview.template.yaml
* https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/family-public-law/fpl-case-service.yaml (and prod)
* service/src/main/resources/bootstrap.yaml
* add the secrets to fpla-aat and fpla-demo vaults, if we have got the final emails we can add them to prod vault. I am happy to quickly show it to you how this is done!

Reply:
I've added to all of above (except for prod)